lib/benchee/formatters/console.ex

```diff
@@ -106,24 +129,24 @@ defmodule Benchee.Formatters.Console do
     |> to_string
   end
 
-  defp label_width(jobs) do
-    max_label_width = jobs
-      |> Enum.map(fn({job_name, _}) -> String.length(job_name) end)
+  defp label_width(scenarios) do
+    max_label_width = scenarios
+      |> Enum.map(&(String.length(&1.job_name)))
       |> Stream.concat([@default_label_width])
       |> Enum.max
     max_label_width + 1
   end
 
-  defp job_reports(jobs, units, label_width) do
-    Enum.map(jobs, fn(job) -> format_jobs job, units, label_width end)
+  defp scenario_reports(scenarios, units, label_width) do
+    Enum.map(scenarios, &(format_scenario(&1, units, label_width)))
   end
 
-  defp units(jobs, %{unit_scaling: scaling_strategy}) do
+  defp units(scenarios, %{unit_scaling: scaling_strategy}) do
     # Produces a map like
     #   %{run_time: [12345, 15431, 13222], ips: [1, 2, 3]}
     measurements =
-      jobs
-      |> Enum.flat_map(fn({_name, job}) -> Map.to_list(job) end)
+      scenarios
+      |> Enum.flat_map(&(Map.to_list(&1.run_time_statistics)))
```

Comment:
Ok I might just go through here later and convert them back to `fn` or you convince me why `&` is great, always happy to learn :)